app/src/test/java/com/x8bit/bitwarden/ui/auth/feature/masterpasswordguidance/MasterPasswordGuidanceScreenTest.kt

```diff
@@ -0,0 +1,74 @@
+package com.x8bit.bitwarden.ui.auth.feature.masterpasswordguidance
+
+import androidx.compose.ui.test.onNodeWithContentDescription
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import com.x8bit.bitwarden.data.platform.repository.util.bufferedMutableSharedFlow
+import com.x8bit.bitwarden.ui.platform.base.BaseComposeTest
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+
+class MasterPasswordGuidanceScreenTest : BaseComposeTest() {
+
+    private var onNavigateBackCalled = false
+    private var onNavigateToGeneratorCalled = false
+    private val mutableEventFlow = bufferedMutableSharedFlow<MasterPasswordGuidanceEvent>()
+    private val viewModel = mockk<MasterPasswordGuidanceViewModel>(relaxed = true) {
+        every { eventFlow } returns mutableEventFlow
+    }
+
+    @Before
+    fun setup() {
+        resetVars()
```

Comment:
Is this needed?

Reply:
technically no, intent was to ensure clean test env per since those fields are mutable. But not over-engineering for the future is also a good rule of thumb. #howtocodewithanxiety

Reply:
I think those can go though, each test should already run independently, right?